### DISCUSSION TITLE
Responsive and/or full width?

### DISCUSSION THREAD

**Discussion by mikealche** [gpbl/react-day-picker#1452] [Support] [answered]
Hey there! 

Thanks for this component! 

I was wondering if it was possible to make it stretch to the width of the parent element?

**Comment by mikealche** [comment, marked as answer]
```

  .rdp-table {
      max-width: none;
      width: 100%;
  }

  .rdp-month {
      width: 100%;
  }
```
Figured it out!

**Comment by naveen1337** [comment]
But stretching calendar by that causes a some problem in date range selection

![image](https://user-images.githubusercontent.com/70107069/203129094-6795b992-8d1e-47dd-867e-e9ff14749ce7.png)

can we fix that, it appears to be some margin from flex container

**Reply by gpbl** [reply]
@naveen1337 investigate with the browser dev tools how the css classes work in DayPicker - in this case, we made the cells to have always the same width – reason of the effect above. By overriding the `.rdp-*` sectors in your CSS, you can change this behavior. 

The original CSS is https://github.com/gpbl/react-day-picker/blob/master/packages/react-day-picker/src/style.css
